Gifski/IntTextField.swift

```diff
@@ -0,0 +1,61 @@
+import AppKit
+import Carbon
+
+final class IntTextField: NSTextField, NSTextFieldDelegate {
+	// Delta used for arrow navigation
+	var delta = 1
+
+	// Delta used for option + arrow navigation
+	var biggerDelta = 10
+
```

Comment:
I was thinking that you could set the minimum and maximum value here and have the textfield itself value those simple cases. I want to make it as reusable as possible.

Reply:
Hmm, the problem here is that the minimum/maximum will change with each type change (pixels/percent). Do you think we should go with min/max still and set it each time we change the type or have a "validate" closure that we could pass to the text field?

Reply:
> Do you think we should go with min/max still and set it each time we change the type

Yeah, I don't see a problem with that. We manually set a lot of properties elsewhere.